content/steps/step-1.md

```diff
@@ -60,23 +74,43 @@ This line does 3 things:
 - It fills in the variable `color` with the word `purple`
 - It allows you to retrieve the word `purple` any time again in your code by reading from the variable `color`
 
-You can choose almost any word that comes to your mind, when naming a variable. It's also possible to include numbers in your variable name. And after a variable has been declared for the first time, you can update its value anytime further down in your code as follows:
+You can choose almost any word that comes to your mind, when naming a variable. It's also possible to include numbers in your variable name.
+
+After a variable has been declared, you can read its value by using its name further down your program:
+
+
+```js
+let color = 'purple';
+
+// ...other code
+
+console.log(color);
+// => prints "purple" to your console
+```
+
+Also, after a variable has been declared for the first time, you can update its value anytime further down in your code as follows:
 
 ```js
+let color = 'purple';
+
+// ...other code
+
 color = 'blue';
 ```
 
 When **updating** an **already declared** variable, you **must not** use the variable keyword. The variable keyword is only used when you declare a new variable for the first time.
 
 Besides that, we also see that each code instruction - a so-called _statement_ - ends with a semi-colon `;`. It acts as like a colon after a phrase indicating to the browser that a full code instruction ends right there.
 
-How can you retrieve the value of a particular variable? We will see how to do so in the next exercise.
+Note: In the examples above we're assigning a word, or a so-called _string_, to our variable `color`. A string describes a number of characters, either letters, numbers or special characters, which are _stringed_ together. A string always needs to be written in quotation marks - you are free to choose either single quotes `'` or double quotes `"`.
+
```

Comment:
TIL: why strings are called like that 😄

Reply:
Yeah, that's at least how I remembered it better :-)